samples/snippets/fsharp/VS_Snippets_CLR_System/system.BitConverter.DoubleInt64/FS/bitstodbl.fs

```diff
@@ -0,0 +1,62 @@
+module bitstodbl
+
+//<Snippet1>
+open System
+
+let print obj1 obj2 = 
+    printfn $"{obj1,20}{obj2,27:E16}"
+
+// Reinterpret the long argument as a double.
+let longBitsToDouble argument =
+    let doubleValue = BitConverter.Int64BitsToDouble argument
+
+    // Display the argument in hexadecimal.
+    print (String.Format("0x{0:X16}", argument)) doubleValue
+
+printfn "This example of the BitConverter.Int64BitsToDouble(Int64) \nmethod generates the following output.\n"
+print "long argument" "double value"
+print "-------------" "------------"
+
+// Convert long values and display the results.
+longBitsToDouble 0
+longBitsToDouble 0x3FF0000000000000L
+longBitsToDouble 0x402E000000000000L
+longBitsToDouble 0x406FE00000000000L
+longBitsToDouble 0x41EFFFFFFFE00000L
+longBitsToDouble 0x3F70000000000000L
+longBitsToDouble 0x3DF0000000000000L
+longBitsToDouble 0x0000000000000001L
+longBitsToDouble 0x000000000000FFFFL
+longBitsToDouble 0x0000FFFFFFFFFFFFL
+longBitsToDouble 0xFFFFFFFFFFFFFFFFL       //
+longBitsToDouble 0xFFF0000000000000L            //
+longBitsToDouble 0x7FF0000000000000L
+longBitsToDouble 0xFFEFFFFFFFFFFFFFL        //
```

Comment:
I'm curious about the trailing `//` on these lines. Is there are purpose?

Reply:
no idea how those got there, I've removed them